tests/std/tests/GH_002992_unwrappable_iter_sent_pairs/test.compile.pass.cpp

```diff
@@ -124,101 +125,73 @@ struct Nontrivial {
-    test_not_unwrappable_pair<test_iter<random, int, wyes>, sent_int>();
-    test_not_unwrappable_pair<test_iter<bidi, int, wyes>, sent_int>();
-    test_not_unwrappable_pair<test_iter<input, int, wyes>, sent_int>();
+    test_not_unwrappable_pair<test::iterator<contiguous, int>, sent_int>();
+    test_not_unwrappable_pair<test::iterator<random, int>, sent_int>();
+    test_not_unwrappable_pair<test::iterator<bidi, int>, sent_int>();
+    test_not_unwrappable_pair<test::iterator<input, int>, sent_int>();
 
-    test_not_unwrappable_pair<test_iter<contiguous, Nontrivial, wyes>, sent_nt>();
-    test_not_unwrappable_pair<test_iter<random, Nontrivial, wyes>, sent_nt>();
-    test_not_unwrappable_pair<test_iter<bidi, Nontrivial, wyes>, sent_nt>();
-    test_not_unwrappable_pair<test_iter<input, Nontrivial, wyes>, sent_nt>();
+    test_not_unwrappable_pair<test::iterator<contiguous, Nontrivial>, sent_nt>();
+    test_not_unwrappable_pair<test::iterator<random, Nontrivial>, sent_nt>();
+    test_not_unwrappable_pair<test::iterator<bidi, Nontrivial>, sent_nt>();
+    test_not_unwrappable_pair<test::iterator<input, Nontrivial>, sent_nt>();
 }
 
 void test_sent_unwrappable() {
-    constexpr auto wyes = test::IsWrapped::yes;
-    constexpr auto wno  = test::IsWrapped::no;
-
     using test::contiguous, test::random, test::bidi, test::input;
 
-    using sent_int = test_sent<int, wyes>;
-    using sent_nt  = test_sent<Nontrivial, wyes>;
+    using sent_int = test::sentinel<int>;
+    using sent_nt  = test::sentinel<Nontrivial>;
 
-    test_not_unwrappable_pair<test_iter<contiguous, int, wno>, sent_int>();
-    test_not_unwrappable_pair<test_iter<random, int, wno>, sent_int>();
-    test_not_unwrappable_pair<test_iter<bidi, int, wno>, sent_int>();
-    test_not_unwrappable_pair<test_iter<input, int, wno>, sent_int>();
+    test_not_unwrappable_pair<test::iterator<contiguous, int>::unwrapping_ignorant, sent_int>();
+    test_not_unwrappable_pair<test::iterator<random, int>::unwrapping_ignorant, sent_int>();
+    test_not_unwrappable_pair<test::iterator<bidi, int>::unwrapping_ignorant, sent_int>();
+    test_not_unwrappable_pair<test::iterator<input, int>::unwrapping_ignorant, sent_int>();
 
-    test_not_unwrappable_pair<test_iter<contiguous, Nontrivial, wno>, sent_nt>();
-    test_not_unwrappable_pair<test_iter<random, Nontrivial, wno>, sent_nt>();
-    test_not_unwrappable_pair<test_iter<bidi, Nontrivial, wno>, sent_nt>();
-    test_not_unwrappable_pair<test_iter<input, Nontrivial, wno>, sent_nt>();
+    test_not_unwrappable_pair<test::iterator<contiguous, Nontrivial>::unwrapping_ignorant, sent_nt>();
+    test_not_unwrappable_pair<test::iterator<random, Nontrivial>::unwrapping_ignorant, sent_nt>();
+    test_not_unwrappable_pair<test::iterator<bidi, Nontrivial>::unwrapping_ignorant, sent_nt>();
+    test_not_unwrappable_pair<test::iterator<input, Nontrivial>::unwrapping_ignorant, sent_nt>();
 }
 
 void test_no_unwrappable() {

```

Comment:
Maybe `test_neither_unwrappable`? (Yeah, I'm getting into nits. Feel free to ignore,)